assets/js/click-to-copy.js

```diff
@@ -0,0 +1,91 @@
+let codeListings = document.getElementsByTagName('pre');
+
+for (let index = 0; index < codeListings.length; index++)
+{
+  if (codeListings[index].parentNode.classList.contains('highlight'))
+   {
+     const codeSample = codeListings[index].querySelector('code');
+     const copyButton = document.createElement("button");
+     copyButton.setAttribute('role', 'button');
+     copyButton.setAttribute('onmousedown', 'copyCode(this)');
```

Comment:
`onmousedown` also captures right-clicks, so we probably want `onclick` instead, same for the tooltip below